manim/cli/render/commands.py

```diff
@@ -136,4 +139,21 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if nov or args["notify_outdated_version"]:
+        manim = requests.get("https://pypi.org/pypi/manim/json")
+        releases = manim.json()["releases"].keys()
+        latest = max(releases)
+
+        if latest != __version__:
+            console.print(
+                f"[yellow]WARNING: You are running on an outdated version of manim[/yellow]"
+            )
+            console.print(f"Installed version: [red]v{__version__}[/red]")
+            console.print(f"Latest version: [green]v{latest}[/green]")
+            console.print(
+                f"[yellow]To disable this warning, go to [white]manim/_config/default.cfg[/white] and set [red]notify_outdated_version[/red] to [/yellow]False"
+            )
+        else:
+            console.print(f"[green]You are using the latest version of manim[/green]")
```

Comment:
No need to inform users they're up to date. Especially because this isn't always true... they would be running the latest released version (e.g. v0.5.0 for now), but the latest is actually whatever is on master
```suggestion
```